pkg/model/model_eks_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	 http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package model_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/aws-controllers-k8s/code-generator/pkg/model"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aws-controllers-k8s/code-generator/pkg/testutil"
+)
+
+func TestEKS_WithNestedReference(t *testing.T) {
+	_ = assert.New(t)
```

Comment:
this doesn't really make sense :) You should do something like this:

```go
assert := assert.New(t)
```

and then later on in the test, you don't need to supply the `t` argument when calling `assert.Equal()`..

Reply:
Done. It was a leftover that i forgot to clean